docs/datadog_monitor.md

```diff
@@ -0,0 +1,134 @@
+# Getting Started
+
+This procedure describes the simplest and fastest way to deploy a `DatadogMonitor` with the Datadog Operator. **Note: Operator version 0.6+ is required.**
+
+## Prerequisites
+
+Using the Datadog Operator requires the following prerequisites:
```

Comment:
This is a bit confusing since it implies that these are the prerequisites for Datadog Operator instead of for DatadogMonitor CRD. Suggesting to change this line to "Install the Datadog Operator in your cluster using the following:"

Reply:
yea good catch, this was from a copy-paste. i'll fix